### PR TITLE
Project listing

### PR DESCRIPTION
**Pull request by eatyourgreens**
Adds a basic project listing, linking each project up to the project contents view.

### CODE REVIEW

File: src/ducks/contents.js [hunk 1]
```diff
@@ -26,12 +26,12 @@ const projectContentsReducer = (state = initialState, action) => {
 };
 
 // Action Creators
-const fetchProjectContents = () => {
+const fetchProjectContents = (project_id) => {
```
Comment (simoneduca) [inline src/ducks/contents.js]: You're requesting `project_contents` with `project_id`, which isn't right. `project_contents` id is needed, which available in project `links`.

Comment (eatyourgreens) [inline src/ducks/contents.js]: My bad, this should actually be requesting a list of all `ProjectContents` resources for a given project, then listing them by language.

Comment (eatyourgreens) [inline src/ducks/contents.js]: Fixed now.

File: src/ducks/contents.js [hunk 2]
```diff
@@ -31,7 +31,10 @@ const fetchProjectContents = (project_id) => {
     dispatch({
       type: FETCH_PROJECT_CONTENTS,
     });
-    apiClient.type('project_contents').get(project_id)
+    const query = {
+      project_id: project_id
```
Comment (simoneduca) [inline src/ducks/contents.js]: Shorthand property is nicer perhaps?

Comment (eatyourgreens) [inline src/ducks/contents.js]: Fixed.

**Comment by simoneduca** [comment]
LGTM!

**Pull request merged**